docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/enforcement.adoc

```diff
@@ -1,44 +1,49 @@
 
+////
 Enforcement configurations scan every commit into your repository and suggest fix remedies, if any violation is detected, this is in addition to the scan that Prisma Cloud periodically performs on your repositories, the results for which are accessible on Projects.
-The periodic scans have a predefined severity threshold which are run across code categories that result in three run rules.
+////
+The scans are applied across code categories, leading to the enforcement of three run rules:
 
-* *Hard Fail*
-+
-A repository scan result fails when Prisma Cloud detects a violation or a vulnerability.
+* *Hard Fail*: A repository scan result fails when Prisma Cloud detects a violation or a vulnerability.
 
-* *Soft Fail*
-+
-A repository scan result is a pass and a notification appears on the console when Prisma Cloud detects a violation or a vulnerability.
+* *Soft Fail*: A repository scan result passes, but a notification is displayed on the console when Prisma Cloud detects a violation or a vulnerability.
 
-* *Comment bot*
-+
-A repository scan result  displays issues with fix suggestions as comments with Pull requests accessible on VCS (Version Control System).
-
-Prisma Cloud scans multiple code categories to identify vulnerabilities and violations unique to the category. See the table for more details.
+* *Comment bot*: A repository scan result displays issues with suggested fixes as comments within Pull Requests on the VCS (Version Control System). 
 
+Prisma Cloud scans the following code categories to detect vulnerabilities and violations unique to each category. 
 
 [cols="1,2", options="header"]
 |===
 |Code Category | Description
 
 |*Vulnerabilities*
-|Vulnerabilities found in open source packages.
-
-|*Licenses*
-|License compliance issues found in open source packages and container images.
+|SCA vulnerabilities found in open source packages
 
 |*Infrastructure as Code (IaC)*
-|Misconfiguration issues found in IaC files (relevant for users who provision and manage their infrastructure via code).
+|Misconfiguration issues found in IaC files
+
+// |*CI/CD Risks*
+// |Identifies vulnerabilities in CI/CD pipelines.
+
+|*Licenses*
+|License compliance issues found in open source packages and container images
 
 |*Secrets*
-|Secret leaks across code files that might hinder access to information, services or assets.
+|Secret leaks across code files that might hinder access to information, services or assets
 
 |===
 
-To understand the default scan parameter on Prisma Cloud with the enforcement run result see the table.
+=== Default Enforcement Rules
+
+The following table displays the default enforcement rules associated with each code category, determined by severity level. These rules apply to all repositories by default.
+
+EXAMPLE
```

Comment:
We don't use EXAMPLE in all caps as a leading sentence in our content. Please remove this and just lead with the sentence, "In the table below ..... Please apply this throughout this topic.

Reply:
Done